### PR TITLE
feat: get identity by public key endpoints

### PR DESCRIPTION
**Pull request by shuplenkov**

### CODE REVIEW

File: lib/abci/handlers/query/identityIdByFirstPublicKeyQueryHandlerFactory.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   */
+  async function identityIdByFirstPublicKeyQueryHandler({ publicKeyHash }) {
+    const identityId = await publicKeyIdentityIdRepository.fetch(publicKeyHash);
+
+    if (!identityId) {
+      throw new NotFoundAbciError('Identity not found');
+    }
+
+    return new ResponseQuery({
+      value: Buffer.from(identityId),
```
Comment (shumkov) [inline lib/abci/handlers/query/identityIdByFirstPublicKeyQueryHandlerFactory.js:29]: `identityId` is base58

File: lib/dpp/DriveStateRepository.js [hunk 2]
```diff
@@ -49,6 +49,15 @@ class DriveStateRepository {
     const transaction = this.getDBTransaction('identity');
 
     await this.identityRepository.store(identity, transaction);
+
+    const storeMapPromises = identity.getPublicKeys()
```
Comment (shumkov) [inline lib/dpp/DriveStateRepository.js:53]: We already have `storePublicKeyIdentityId`

File: test/unit/dpp/DriveStateRepository.spec.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
 const getDocumentsFixture = require('@dashevo/dpp/lib/test/fixtures/getDocumentsFixture');
```
Comment (shumkov) [inline test/unit/dpp/DriveStateRepository.spec.js:1]: We need to revert it back

File: lib/dpp/DriveStateRepository.js (hunk 2 above)
Comment (shuplenkov) [reply lib/dpp/DriveStateRepository.js:53]: Why don't you want to do it in another PR?

File: test/unit/dpp/DriveStateRepository.spec.js (hunk 3 above)
Comment (shuplenkov) [reply test/unit/dpp/DriveStateRepository.spec.js:1]: Revert to what? I haven't change it

File: lib/dpp/DriveStateRepository.js (hunk 2 above)
Comment (shumkov) [reply lib/dpp/DriveStateRepository.js:53]: We don't need this logic. It's already implemented in `storePublicKeyIdentityId` method.

File: test/unit/dpp/DriveStateRepository.spec.js (hunk 3 above)
Comment (shumkov) [reply test/unit/dpp/DriveStateRepository.spec.js:1]: Why I see changes in this file then :))

**Pull request merged**